src/components/DrumElement.tsx

```
@@ -0,0 +1,106 @@
+import { useEffect, useState } from 'react'
+import styled, { CSSProperties } from 'styled-components'
+
+interface ElementProps {
+  src?: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)